hyperbolic/euclid/Arc.py

```diff
@@ -40,6 +40,17 @@ def reverse(self):
     def reversed(self):
         return Arc(self.cx, self.cy, self.r, self.endDeg, self.startDeg,
                    cw=not self.cw)
+    def isPointOnSegment(self, x, y):
+        '''Assumes that Point is on the circle'''
```

Comment:
When adding code to an existing project, it is best to try to match the same code style.  I started this project a while ago so the style doesn't conform to [PEP8](https://pep8.org/) unfortunately.
```suggestion
        ''' Assumes that Point is on the circle '''
```

Thanks for adding docstrings.  Consider making them more descriptive for users who haven't looked at the code.
```suggestion
        ''' Assumes that the given point is on the circle and returns True if on
        this arc '''
```